odhargo/README.md

```diff
@@ -1,37 +1,49 @@
-# Argo 
+# ODH Argo
+
+![ODH Argo version](https://img.shields.io/badge/ODH_Argo_version-v2.11.7-yellow.svg) <!-- v2.11.7 -->
+![Upstream version](https://img.shields.io/github/v/release/argoproj/argo?label=Upstream%20release)
+
+ODH Argo component installs Argo that is namespace bound and not cluster wide. There are two pods running after installation
 
-Argo component installs Argo version 2.7 that is namespace bound and not cluster wide. There are two pods running after installation
 1. Argo Server
 2. Argo Controller
 
 This Argo installation uses the "k8sapi" executor to work on Openshift.
 
 ### Folders
-There is only one folder, `base`. This folder includes the installation yaml files to all necessary Argo resources that need to be installed
 
-### Argo Portal
+1. `cluster` folder contains all the cluster wide resources required to be installed by the operator before Argo can be deployed. It contains all `CustomResourceDefinitions` and `ClusterRoles` which aggregates permissions for those CRDs to admin/edit/view project roles.
+
+2. There's only a single `base` folder within the `odhargo` folder. This `base` includes all installation yaml files to all necessary namespaced Argo resources.
 
-This installation creates a route to the Argo portal. To access the portal go to `Routes` and click on the `Argo Portal` route.
+### Argo Server
 
+This installation creates a route to the Argo portal. To access the portal go to `Routes` and click on the `Argo Server` route.
 
 ### Installation
+
 To install Argo add the following to the `kfctl` yaml file.
 
-```
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: odhargo
-    name: odh-argo
+```yaml
+- kustomizeConfig:
+    repoRef:
+      name: manifests
+      path: odhargo
+  name: odh-argo
 ```
 
 ### Run a Workflow
-To run an example workflow, you can use the portal UI to run the example workflow, however the `serviceAccountName` has to be added to the workflow as shown below. We created this example `serviceAccount` as part of the installation to give basic permissions to a workflow. You can edit this `serviceAccount` and add more pemissions if your workflow requires that.
```

Comment:
Does this updated version of Argo remove the need a service account with elevated privileges to allow for advanced Workflow features (artifact passing) that fail(ed) with the `default` serviceAccount?

Reply:
When it comes to artifact passing access to pods and logs is all Argo needs. So with just these permissions Argo Workflows should work just fine.

We should probably link the upstream docs in here though:
- https://argoproj.github.io/argo/service-accounts/
- https://argoproj.github.io/argo/workflow-rbac/

Mind that Argo uses service accounts to enforce proper RBAC on workflows (different SA is used to execute the workflow for different user roles), so ODH should not be much opinionated here.
- https://argoproj.github.io/argo/argo-server-sso/#sso-rbac

Reply:
Documenting this from an offline conversation, we want components to be standalone and preserve the default permissions for the `default` ServiceAccount.

Reply:
I've reintroduced the `argo-workflow` SA, removed all changes to the `default` SA and assinged `argo-workflow` SA as the default for Workflow execution. WDYT? :slightly_smiling_face: